projects/adrena/index.js

```diff
@@ -0,0 +1,51 @@
+const { sumTokens2, getConnection } = require("../helper/solana");
+const { PublicKey } = require("@solana/web3.js")
+const { decodeAccount } = require('../helper/utils/solana/layout')
+const ADDRESSES = require('../helper/coreAssets.json')
+
+async function staking() {
```

Comment:
you can simplify it a bit, `return sumTokens2({ tokenAccounts: ['9nD5AenzdbhRqWo7JufdNBbC4VjZ5QH7jzLuvPZy2rhb']})`